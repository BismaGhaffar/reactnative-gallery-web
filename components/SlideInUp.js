import styled, { keyframes } from 'styled-components'
import { slideInUp } from 'react-animations'

const slide = keyframes`${slideInUp}`

const SlideInUp = styled.div`
  animation: ${props => props.timer || 1}s ${props => props.delay || 0}ms
    ${slide};
`

export default SlideInUp
