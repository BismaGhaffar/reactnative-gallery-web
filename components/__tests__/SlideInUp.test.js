import React from 'react'
import renderer from 'react-test-renderer'
import SlideInUp from '../SlideInUp'

it('SlideInUp can be created', () => {
  const comp = renderer.create(<SlideInUp />)
  expect(comp).toBeDefined()
})

it('<SlideInUp /> toMatchSnapshot', () => {
  const tree = renderer.create(<SlideInUp />).toJSON()
  expect(tree).toMatchSnapshot()
})
