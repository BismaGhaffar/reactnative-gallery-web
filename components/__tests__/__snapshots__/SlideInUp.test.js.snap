// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SlideInUp /> toMatchSnapshot 1`] = `
<div
  className="SlideInUp-s178605u-0 dTWHXn"
/>
`;
